src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -6,6 +6,8 @@
 #include "pch.h"
 #include "NonClientIslandWindow.h"
 
+extern double NON_CLIENT_DRAGBAR_WIDTH;

```

Comment:
This I don't like. Shouldn't this be looked up by a reference to another object and calling a getter instead of effectively declaring a global of sorts?

Reply:
This is obtained from the TerminalApp instance